config/settings.yaml.tpl

```diff
@@ -19,9 +19,18 @@ default:
     gateway:  # More info at GATEWAYS.md
       default:
         kind: "SystemApicast"
+        httpbin:
```

Comment:
This should 100% be under WASMGateway section, it doesn't make sense for SystemApicast and for other gateways

Reply:
Addressed